web/themes/joinup/templates/search_api/facets-type-wrapper--more.html.twig

```diff
@@ -12,8 +12,13 @@
  * @ingroup themeable
  */
 #}
+{% if items|length > 2 %}
+  {% set class='tab__container tab__container--wide' %}
```

Comment:
@kalinchernev, I would be grateful if you could check if this what I have pushed is what we are looking for. Regarding desktop, there is also possibility to achieve this:

![screen shot 2017-08-03 at 20 04 58](https://user-images.githubusercontent.com/19408231/28936392-e6c1dd40-7887-11e7-9f0f-293dd011849d.png)

Which version (the one which I pushed or the one from ss) is closer to your requirements?